api/src/test/java/org/openmrs/api/OrderServiceTest.java

```diff
@@ -1414,22 +1414,20 @@ public void retireOrderFrequency_shouldRetireGivenOrderFrequency() throws Except
 	@Test
 	@Verifies(value = "should unretire given order frequency", method = "unretireOrderFrequency(OrderFrequency)")
 	public void unretireOrderFrequency_shouldUnretireGivenOrderFrequency() throws Exception {
-		OrderFrequency orderFrequency = orderService.getOrderFrequency(3);
+		executeDataSet("org/openmrs/api/include/OrderServiceTest-otherOrderFrequencies.xml");
+		OrderFrequency orderFrequency = orderService.getOrderFrequency(103);
 		assertNotNull(orderFrequency);
 		assertTrue(orderFrequency.isRetired());
```

Comment:
in the standard data set concept_class_id="19" has been set for order Frequency (https://github.com/openmrs/openmrs-core/blob/9b977a18fcc759af9ad782110b18efe27175dd76/api/src/test/resources/org/openmrs/include/standardTestDataset.xml#L35)   and the backing concept (L172) for order frequency 3 (L342 of standardTestDataset.xml) has not been assigned this class_id because of which a validation exception was caused.
In the new dataset a new concept with new concept id has been created that has class_id ="19". This is the backing concept for orderFrequency 103 that i have switched to